composer.json

```diff
@@ -16,15 +16,21 @@
     ],
     "minimum-stability": "dev",
     "autoload": {
-        "files": [ "command.php" ]
+        "psr-4": {
+            "WP_CLI\\": "src/WP_CLI"
```

Comment:
This is actually PSR-0, not PSR-4.

With PSR-0, you add the entire namespace as a folder structure:

```
Vendor\Package\ClassA => src/Vendor/Package/ClassA.php
Vendor\Package\SubModule\ClassB => src/Vendor/Package/SubModule/ClassB.php
```

With PSR-4, you only use the shortest "relative" namespace as a folder structure:
```
Vendor\Package\ClassA => src/ClassA.php
Vendor\Package\SubModule\ClassB => src/SubModule/ClassB.php
```

The goal of PSR-4 is to keep the individual packages as clean as possible in terms of folder structure.
So, here, this would be:
```
"psr-4": { "WP_CLI\\": "src" }
WP_CLI\CommandClass => src/CommandClass.php
```